resources/views/threads/_list.blade.php

```diff
@@ -5,6 +5,9 @@
                 <div class="flex">
                     <h4>
                         <a href="{{ $thread->path() }}">
+                            @if($thread->pinned)
```

Comment:
I would put a space after `@if` and before `($thr....`

Reply:
Done 👍